cgi/ingredients.pl

```diff
@@ -29,6 +29,13 @@
 my $code = normalize_code(param('code'));
 my $id = param('id');
 
+my $ocr_engine = param('ocr_engine');
+
+if (not defined $ocr_engine) {
+	$ocr_engine = "tesseract";
+	# $ocr_engine = "google_cloud_vision";
```

Comment:
Intentional?

Reply:
Yes. I used it when developing, and I'm leaving the comment here to show what values are possible.